README.md

```diff
@@ -2,31 +2,192 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Continue learning about conditionals.
+- Take a look at `if` statements inside of `if` statements.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+In the previous lesson, we introduced a chained conditional statements using the
+`else if` statement. Stated before, we said that an `if` block could hold more
+than one statement. But what if we want one of those statements to be another
+`if` statement? Is that possible?
 
-## Topic 1
+The answer is yes! Since `if` statements can contain multiple statements inside
+their `if` block, they can also contain other `if` statements. This is called
+**nested conditionals**.
```

Comment:
Consider warning students that too much nesting can hurt the brains of code reviewers.

Reply:
Thanks!